### DISCUSSION TITLE
Dedicated page to address all common pitfalls of the library usage

### DISCUSSION THREAD

**Discussion by slainless** [chakra-ui/panda#2664] [Ideas]
### Subject

Documentation request

### Description

Coming from personal experience as pandacss newcomer, it's easier to fall to common problems of the library usage, most of the time caused by the misunderstanding of the nature of the library (static generation). This will be more apparent when using preset such as Park UI where it's expected for the end user to extends and/or create derivative components. More often, I was left wondered why the style is not generated/applied as expected and have to skim multiple pages from "Recipes" to "JSX Style Props" etc. just to find what caused this problem.

I suggest to create a dedicated page that aggregates/gathers all common pitfalls of library usage to ease library usage and adoption, something like [React docs' "Rules" page](https://react.dev/reference/rules).

**Comment by astahmer** [comment]
maybe https://panda-css.com/docs/overview/faq ? would you like to contribute ?
